test/suites/snapshots.sh

```diff
@@ -1,23 +1,26 @@
 test_snapshots() {
-  snapshots
+  snapshots "lxdtest-$(basename "${LXD_DIR}")"
```

Comment:
for future reference changes like this that arent related to the PR's main change should be at least in their own commits so they can be backported. By doing drive-by improvements in the same commit as feature work you're making it harder for these improvements to be backported to the LTS branches without the feature work.

Reply:
Yes makes sense, let me put this into another commit for the sake of completeness :)